lib/IRGen/GenMeta.cpp

```diff
@@ -4216,7 +4216,7 @@ SpecialProtocol irgen::getSpecialProtocolID(ProtocolDecl *P) {
   case KnownProtocolKind::Encodable:
   case KnownProtocolKind::Decodable:
   case KnownProtocolKind::StringInterpolationProtocol:
-  case KnownProtocolKind::Expression:
+  case KnownProtocolKind::Differentiable:
```

Comment:
This is still a tensorflow branch-specific addition. Until upstreaming is finished, I think this should stay within the `SWIFT_ENABLE_TENSORFLOW` range.

Reply:
I see.

My thought was: upstreaming PRs don't use `SWIFT_ENABLE_TENSORFLOW`. After the upstreaming PR https://github.com/apple/swift/pull/27511 is merged, we'll need to choose where to place this currently `SWIFT_ENABLE_TENSORFLOW` addition.

I chose to match the code locations https://github.com/apple/swift/pull/27511 to avoid merge conflicts. I'm also fine with keeping the old location (not matching code locations) as long as we standardize. What do you think?

Reply:
I don't have a strong preference. Given that we have an upstreaming PR, it would be fine to match the code location.

Reply:
Works for me.